services/blockchain-indexer/shared/dataService/business/network.js

```diff
@@ -79,24 +95,28 @@ const getPeers = async params => {
 	};
 };
 
-const getConnectedPeers = async params => {
-	const response = await getPeers(Object.assign(params, { state: 'connected' }));
+const getNetworkConnectedPeers = async params => {
+	const response = await getNetworkPeers(Object.assign(params, { state: 'connected' }));
```

Comment:
maybe we can define a `PEER_STATE` constant like below?

```
PEER_STATE = {
CONNECTED: 'connected',
DISCONNECTED: 'disconnected',
}
```

Reply:
Done!